front/app/modules/commercial/insights/admin/containers/Reports/charts/HorizontalBarChartWithoutStream.tsx

```diff
@@ -93,8 +82,7 @@ export const HorizontalBarChartWithoutStream: React.SFC<
           y={y}
           dx={30}
           dy={-6}
-          fill={chartLabelColor}
-          fontSize={chartCategorySize}
```

Comment:
I am removing `fontSize` here since `chartCategorySize` never even existed, so it was always `undefined`